Gemfile

```diff
@@ -4,7 +4,9 @@ gem "jbuilder", "~> 2.5"
 gem "pg"
 gem "puma", "~> 3.7"
 gem "rails", "~> 5.1.4"
+gem "responders"
```

Comment:
The responders gem is added but the install generator hasn't been run.
For future reference, I would add the gem, run the install command, and encapsulate those changes in one commit.

Unless the gem is being used, there is no reason to add it.